pkg/apis/cnsoperator/config/cnsfileaccessconfig_crd.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: cnsfileaccessconfigs.cns.vmware.com
+spec:
+  group: cns.vmware.com
+  names:
+    kind: CnsFileAccessConfig
+    listKind: CnsFileAccessConfigList
+    plural: cnsfileaccessconfigs
+    singular: cnsfileaccessconfig
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      description: CnsFileAccessConfig is the Schema for the cnsfileaccessconfig API
+      properties:
+        apiVersion:
+          description: 'APIVersion defines the versioned schema of this representation
+            of an object. Servers should convert recognized schemas to the latest
+            internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#resources'
+          type: string
+        kind:
+          description: 'Kind is a string value representing the REST resource this
+            object represents. Servers may infer this from the endpoint the client
+            submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/api-conventions.md#types-kinds'
+          type: string
+        metadata:
+          type: object
+        spec:
+          type: object
+          description: CnsFileAccessConfigSpec defines the desired state of CnsFileAccessConfig
+          properties:
+            pvcName:
+              description: PvcName indicates the name of the PVC on the supervisor Cluster.
+              type: string
+              pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$'
+            vmName:
+              description: VMName is the name of VirtualMachine instance on SV cluster.
+              type: string
+              pattern: '^[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*$'
+          required:
+          - volumeName
+          type: object
+        status:
+          type: object
+          description: CnsFileAccessConfigStatus defines the observed state of CnsFileAccessConfig.
+          properties:
+            done:
+              description: Done indicates whether the ACL has been configured on file volume.
+              type: bool
+            accessPoints:
+              description: Access points per protocol supported by the volume.
+              additionalProperties:
+                items:
+                  type: string
```

Comment:
This should be `type: Object` , no ?
In `pkg/apis/cnsoperator/cnsfileaccessconfig/v1alpha1/cnsfileaccessconfig_types.go` accessPoints is a map`AccessPoints map[string]string`

Reply:
Items are string type.